### PR TITLE
Scrub Remaining Andculture references

### PR DESCRIPTION
**Pull request by myty**
Handles #55

- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
-[![All Contributors](https://img.shields.io/badge/all_contributors-18-orange.svg?style=flat-square)](#contributors-)
-
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
-
-[Supplemental Documentation](https://andculturecode.github.io/RSM.HCD.CSharp)
+[![codecov](https://codecov.io/gh/rsm-hcd/RSM.HCD.CSharp/branch/main/graph/badge.svg)](https://codecov.io/gh/rsm-hcd/RSM.HCD.CSharp)
+
+<!--ts-->
+
+-   [RSM.HCD.CSharp](#rsmhcdcsharp)
```
[the remark this answers is not in the document]
Comment (myty) [reply README.md:8]: yeah.  not sure which vscode extensions keeps adding it in.

File: README.md [hunk 2]
```diff
@@ -17,25 +38,21 @@ $: npx cross-env CURRENT_BRANCH=main USE_SSH=true GIT_USER={GITHUB_USERNAME} GIT
...
+### RSM.HCD.CSharp.Core
 
-Commonly used interfaces, patterns and utilities by andculture engineering
+Commonly used interfaces, patterns and utilities by the HCD engineering team at RSM
 
-#### Getting Started (AndcutlureCode.CSharp.Core)
+#### Getting Started (v.CSharp.Core)
 
-This package is installed via NuGet
+This package is installed via NuGetRSM.HCD
```
[the remark this answers is not in the document]
Comment (myty) [reply README.md:47]: Yes.  Good catch!

**Comment by myty** [comment]
> Added some minor fixes, but nothing to hold you up. There are a bunch of obsolete warnings that may need to be handled. I also noticed weird identing in `src/Conductors/RepositoryConductor.cs` that may be a result of mixed tabs and spaces.

Yeah.  We have a bunch of GH Issues to address the obsolete warnings already.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jstevenson) [reply README.md]: If it is the one referenced in https://github.com/rsm-hcd/HCD-Code/blob/main/code-conventions/code-conventions-markdown.md#table-of-contents, then you need the VS Code settings `"markdown-toc.depthFrom": 2,`.

**Review by jstevenson — APPROVED** [review]
Looks great!

**Pull request merged**